pkg/machines/app.jsx

```diff
@@ -59,13 +109,32 @@ class App extends React.Component {
                 { config.provider.name === 'LibvirtDBus' && this.state.activeTab == 1 &&
                 <AggregateStatusCards networks={networks} storagePools={storagePools} changeActiveList={this.changeActiveList} />
                 }
+                {this.state.notifications.length > 0 &&
+                <React.Fragment>
```

Comment:
This fragment wrapper is not needed (but harmless).

Reply:
Removed.